api/features/tests/test_views.py

```diff
@@ -787,6 +789,65 @@ def test_get_feature_evaluation_data(project, feature, environment, mocker, clie
     )
 
 
+def test_create_segment_override_forbidden(feature, segment, environment, api_client):
+    # Given
+    url = reverse(
+        "api-v1:environments:create-segment-override",
+        args=[environment.api_key, feature.id],
+    )
+    user = FFAdminUser.objects.create(email="jim@example.com")
+    api_client.force_authenticate(user)
```

Comment:
This should just use the `staff_client` now, right?

Reply:
Yep! Old code, nice catch!